DSCResources/MSFT_xRunbookDirectory/MSFT_xRunbookDirectory.schema.mof

```diff
@@ -0,0 +1,10 @@
+[ClassVersion("1.0.0.0"), FriendlyName("RunbookDirectory")]
+class MSFT_xRunbookDirectory : OMI_BaseResource
+{
+    [Key, Description("Path to Runbook(s) to be imported. Accepts wildcards.")] String RunbookPath;
+    [Write, Description("Describes the published state of the imported Runbook(s). Defaults to true, publishing the Runbook.")] Boolean Publish;
```

Comment:
Could this follow the Ensure Pattern
 [Write, ValueMap{"Published", "Absent"}, Values{"Published", "Absent"}, Description("...")] String Ensure;